code/modules/mob/living/carbon/human/life/handle_chemicals_in_body.dm

```diff
@@ -5,6 +5,10 @@
 
 	if(reagents)
 
+		if(locked_to && istype(locked_to,/obj/structure/bed/nest))//alien nest will keep their victims alive for a bit longer.
```

Comment:
Have nests do you this you silly silly person, don't modify general procs for interactions with specific objects